runtime/common/src/precompile/schedule_call.rs

```diff
@@ -13,17 +13,25 @@ use frame_support::{
 use module_evm::{Context, ExitError, ExitSucceed, Precompile};
 use module_support::TransactionPayment;
 use primitives::{evm::AddressMapping as AddressMappingT, Balance, BlockNumber};
-use sp_core::{H160, U256};
+use sp_core::H160;
 use sp_std::{convert::TryFrom, fmt::Debug, marker::PhantomData, prelude::*, result};
 
-use super::input::{Input, InputT, BALANCE_BYTES, PER_PARAM_BYTES};
-use codec::Encode;
+use super::input::{Input, InputT, PER_PARAM_BYTES};
+use codec::{Decode, Encode};
 use pallet_scheduler::TaskAddress;
 
 parameter_types! {
 	pub storage EvmSchedulerNextID: u32 = 0u32;
 }
 
+#[derive(Debug, PartialEq, Encode, Decode)]
+struct TaskInfo {
+	id: Vec<u8>,
```

Comment:
```suggestion
	prefix: &'static str,
	id: u32,
```

to avoid another encode